### PR TITLE
Legal links

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Create menu in ShellBar
- Move legal links
- Add get help links to config

**Related issue(s)**

### CODE REVIEW

File: tests/integration/tests/cluster/test-cluster-configuration.spec.js [hunk 1]
```diff
@@ -74,6 +74,11 @@ context('Test Cluster configuration', () => {
 
     cy.get('[title="Profile"]').click();
 
+    cy.get('ui5-li:visible')
+      .contains('Preferences')
+      .invoke('show')
```
Comment (chriskari) [inline tests/integration/tests/cluster/test-cluster-configuration.spec.js:79]: do we need this `.invoke('show')` for the test to pass?
at least locally it works for me without it

File: tests/integration/tests/cluster/test-dashboard-version.spec.js [hunk 2]
```diff
@@ -8,11 +8,18 @@ context('Test Kyma Dashboard Version link', () => {
...
+      .contains('Legal Information')
+      .invoke('show')
+      .click({ force: true });
+
+    cy.get('ui5-responsive-popover[placement-type="Right"]')
+      .get('ui5-li:visible')
       .last()
-      .should('have.attr', 'title')
-      .and('include', 'Kyma Dashboard version:');
+      .invoke('text')
```
Comment (chriskari) [inline tests/integration/tests/cluster/test-dashboard-version.spec.js:22]: same question here for 
`.invoke('show')` and `.invoke('text')`

File: tests/integration/tests/namespace/test-protected-resources.spec.js [hunk 3]
```diff
@@ -127,6 +127,11 @@ context('Test Protected Resources', () => {
   it('Change protection setting', () => {
     cy.get('[title="Profile"]').click();
 
+    cy.get('ui5-li:visible')
+      .contains('Preferences')
+      .invoke('show')
```
Comment (chriskari) [inline tests/integration/tests/namespace/test-protected-resources.spec.js:132]: here as well

File: tests/integration/tests/namespace/test-settings.spec.js [hunk 4]
```diff
@@ -87,6 +102,11 @@ context('Test app settings and preferences', () => {
   it('Shows hidden namespaces', () => {
     cy.get('[title="Profile"]').click();
 
+    cy.get('ui5-li:visible')
+      .contains('Preferences')
+      .invoke('show')
+      .click({ force: true });
```
Comment (chriskari) [inline tests/integration/tests/namespace/test-settings.spec.js:108]: in this file as well

File: tests/integration/tests/namespace/test-resource-validation.spec.js [hunk 5]
```diff
@@ -118,6 +128,11 @@ context('Test resource validation', () => {
 
     cy.get('[title="Profile"]').click();
 
+    cy.get('ui5-li:visible')
+      .contains('Preferences')
+      .invoke('show')
```
Comment (chriskari) [inline tests/integration/tests/namespace/test-resource-validation.spec.js:133]: in this file as well

File: src/header/Header.tsx [hunk 6]
```diff
@@ -52,6 +67,33 @@ export function Header() {
...
 
+  const openNewWindow = (link: string) => {
+    const newWindow = window.open(link, '_blank', 'noopener, noreferrer');
+    if (newWindow) newWindow.opener = null;
+  };
+
+  const handleMenuItemClick = (
+    e: Ui5CustomEvent<MenuDomRef, MenuItemClickEventDetail>,
+  ) => {
+    console.log('clicked', e.detail.text);
```
Comment (chriskari) [inline src/header/Header.tsx:78]: remove console log

File: src/header/Header.tsx [hunk 7]
```diff
@@ -74,30 +116,84 @@ export function Header() {
...
-            icon="customer"
-            colorScheme="Accent10"
-            accessibleName="Preferences"
-          />
+          <>
+            <Avatar
+              icon="customer"
+              colorScheme="Accent6"
+              accessibleName="Preferences"
+              id={'openShellbarMenu'}
```
Comment (chriskari) [inline src/header/Header.tsx:124]: just to make it look more beautiful to me remove the {} brackets 😄

File: tests/integration/tests/cluster/test-cluster-configuration.spec.js (hunk 1 above)
Comment (akucharska) [reply tests/integration/tests/cluster/test-cluster-configuration.spec.js:79]: Form me without `.invoke('show')` it couldn't find `li`

File: tests/integration/tests/namespace/test-protected-resources.spec.js (hunk 3 above)
Comment (akucharska) [reply tests/integration/tests/namespace/test-protected-resources.spec.js:132]: Form me without `.invoke('show')` it couldn't find `li`

File: tests/integration/tests/namespace/test-settings.spec.js (hunk 4 above)
Comment (akucharska) [reply tests/integration/tests/namespace/test-settings.spec.js:108]: Form me without `.invoke('show')` it couldn't find `li`

File: tests/integration/tests/namespace/test-resource-validation.spec.js (hunk 5 above)
Comment (akucharska) [reply tests/integration/tests/namespace/test-resource-validation.spec.js:133]: Form me without `.invoke('show')` it couldn't find `li`

File: tests/integration/tests/cluster/test-cluster-configuration.spec.js (hunk 1 above)
Comment (akucharska) [reply tests/integration/tests/cluster/test-cluster-configuration.spec.js:79]: Yesterday without `.invoke('show')` it failed at finding `li`, but today it somehow work. So I'll remove all the `.invoke('show')`

File: tests/integration/tests/cluster/test-dashboard-version.spec.js (hunk 2 above)
Comment (akucharska) [reply tests/integration/tests/cluster/test-dashboard-version.spec.js:22]: `.invoke('show')` can be removed but `.invoke('text')` is necessary to get Kyma Dashboard version: as string

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
is it possible to make some space between text and arrow?
image

**Review by mrCherry97 — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**